### ISSUE TITLE
Give radio buttons in sign up selector a `<fieldset>`

### ISSUE THREAD

**Issue by emmastephenson** [CDCgov/prime-simplereport#4016]
## Area or Component
Sign up form - selector

## Quick Summary
The three radio buttons aren't labelled in a group, so screenreader users aren't getting the benefit of the labels at the top of the page.

## Accessibility Violation
Several reasons:
- The group label is not associated with its group of radio buttons. The purpose of individual radio buttons in the group cannot be understood without the group label. https://cdc-devtools.dequecloud.com/rule-help/en/label-group-radio-not-associated
- Hidden or empty element receives focus https://cdc-devtools.dequecloud.com/rule-help/en/focus-on-hidden-item
- The visual focus indicator is missing altogether. https://cdc-devtools.dequecloud.com/rule-help/en/focus-indicator-missing

## Action Requested
Give the group of radio buttons a `<fieldset>` matching the top hint text.

## Screenshots (optional)
![Screen Shot 2022-07-20 at 3.42.52 PM.png](https://images.zenhubusercontent.com/604699209bfd9306195da0d7/a42a7feb-2e88-464b-a55e-d9e47c42f69c)

## Original Deque Issue
https://docs.google.com/spreadsheets/d/1b6xjubAN5Uw11DV4_B5zvSu7HeDI6Z-DpV4DMcToqAY/edit#gid=543263785&range=8:8
